cmd/deployctl/cmd/build.go

```diff
@@ -39,6 +39,7 @@ func CollectCmdRun(cmd *cobra.Command, args []string) {
 	var noInterfaceDefaults bool
 	var normalizeConsole bool
 	var noCACertificates bool
```

Comment:
The noCACertificates is now true by default in this implementation, can we deprecate this argument, and return an error message indicating the deprecation of this argument?

Reply:
I have added code to deprecated "noCACertificates"